packages/common/test/models/models.test.ts

```diff
@@ -192,6 +193,8 @@ describe("model utils:", () => {
       )[0] as unknown as portalModule.IUpdateItemOptions;
 
       expect(opts.authentication).toBe(MOCK_AUTH);
+      expect(opts.params).toBeDefined();
```

Comment:
add something that checks that description is still passed through to the http request